app/jobs/in_person/send_proofing_notification_job.rb

```diff
@@ -0,0 +1,107 @@
+          enrollment_code: enrollment.enrollment_code,
+          enrollment_id: enrollment.id,
+        )
+      if enrollment.passed? || enrollment.failed?
+        # send notification and log result
+        phone = enrollment.notification_phone_configuration.formatted_phone
+        message = notification_message(enrollment: enrollment)
+        response = Telephony.send_notification(
+          to: phone, message: message,
+          country_code: Phonelib.parse(phone).country
+        )
+        handle_telephony_result(enrollment: enrollment, phone: phone, telephony_result: response)
+        if response.success?
+          # if notification sent successful
+          enrollment.update(
+            notification_sent_at: Time.zone.now,
+          )
+          # delete notification phone configuraiton if success
+          enrollment.notification_phone_configuration.destroy!
+        end
+      end
+      return true
+    ensure
+      unless enrollment.present?
+        enrollment = InPersonEnrollment.find(
+          enrollment_id,
+          include: [:notification_phone_configuration,
+                    :user],
+        )
+      end
+      analytics(user: enrollment.user).
+        idv_in_person_usps_proofing_results_notification_job_completed(
+          enrollment_code: enrollment.enrollment_code, enrollment_id: enrollment.id,
+        )
+    end
+
+    private
+
+    def handle_telephony_result(enrollment:, phone:, telephony_result:)
+      if telephony_result.success?
+        analytics(user: enrollment.user).
+          idv_in_person_usps_proofing_results_notification_sent_attempted(
+            success: true,
+            enrollment_code: enrollment.enrollment_code,
+            enrollment_id: enrollment.id,
+            telephony_result: telephony_result,
+          )
+        # log success
+      elsif telephony_result.error.is_a?(Telephony::OptOutError)
+        analytics(user: enrollment.user).
+          idv_in_person_usps_proofing_results_notification_sent_attempted(
+            success: false,
+            enrollment_code: enrollment.enrollment_code,
+            enrollment_id: enrollment.id,
+            telephony_result: telephony_result,
+          )
+        PhoneNumberOptOut.mark_opted_out(phone)
+      else
+        analytics(user: enrollment.user).
+          idv_in_person_usps_proofing_results_notification_sent_attempted(
+            success: false,
+            enrollment_code: enrollment.enrollment_code,
+            enrollment_id: enrollment.id,
+            telephony_result: telephony_result,
+          )
+      end
+    end
+
+    def notification_message(enrollment:)
+      proof_date = enrollment.proofed_at ? enrollment.proofed_at.strftime('%m/%d/%Y') : 'NA'
+      I18n.t(
+        'telephony.confirmation_ipp_enrollment_result.sms',
+        app_name: APP_NAME,
+        proof_date: proof_date,
+      )
+    end
+
+    def analytics(user: AnonymousUser.new)
```

Comment:
seems like we always pass in `enrollment.user` all these analytics calls in this file, I think we can remove the Anonymous user default, and maybe update this to take in the enrollment directly?

Reply:
@zachmargolis  passed in user.